src/css/style.css

```diff
@@ -182,5 +182,5 @@ body {
   text-align: center;
```

Comment:
- [ ] Your application is showing a blank screen and therefore, the styling of your application's UI does not match the provided [design template](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Kindly ensure your design follows the design template in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Kindly note that the video has more features than the current project, and therefore you do not need to style all the features implemented in the video. 

- [ ] Your webpack is not properly configured. Kindly update your webpack configuration to look something like this:


const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  entry: './src/index.js',
  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
  devtool: 'inline-source-map',
  devServer: {
    static: './dist',
  },
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
    clean: true,
  },
  mode: 'development',
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: ['style-loader', 'css-loader'],
      },
    ],
  },
};`


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/134218832-b8df2741-aec5-4c3d-9cc9-423a49f35c8b.png)

Reply:
This issue is well resolved! ✔️